app/frontend/src/components/admin/Clients.vue

```diff
@@ -106,10 +150,44 @@ export default {
       this.alertType = typ;
       this.alertMessage = msg;
       this.loading = false;
+    },
+    clicked(value) {
+      this.expanded.push(value);
     }
   },
   mounted() {
     this.getData();
   }
 };
 </script>
+
+<style scoped>
+.kc-search {
+  width: 100%;
+  max-width: 20em;
+  float: right;
+}
+.kc-table{
+  clear: both;
+}
+.kc-table >>> tr.v-data-table__expanded.v-data-table__expanded__row td {
+  border-bottom: 0 !important;
+}
+.kc-table >>> tr.v-data-table__expanded.v-data-table__expanded__content {
+  -webkit-box-shadow: none !important;
+  box-shadow: none !important;
+}
+.kc-table tr.last td {
+  border-bottom: 1px !important;
+  padding-bottom: 10px;
+}
+.kc-nested-table {
```

Comment:
i have some layout issues complicated by vuetify.
i may need to re-think the layout of the expanded row. will check with Mina